modules/cache/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable "project" {
+  description = "Name of the project"
+}
+
+variable "environment" {
+  description = "The environment (development/staging/production)"
+}
+
+variable "cache_store" {
+  description = "Cache store - redis or memcached"
+  type        = string
+  default     = "memcached"
```

Comment:
How about we add input validation to ensure it is a correct value, then we can remove the third ternary option in the outputs file because we don't need to worry about an option that's not recognized.
For example:
```
  validation {
    condition     = contains(["redis", "memcached"], var.cache_store)
    error_message = "Allowed values for cache_store are \"redis\", \"memcached\"."
  }
```

Also, let's remove the default here. They should be specifying it explicitly.